modules/snooker/src/test/ts/browser/CopySelectedTableWidthTest.ts

```diff
@@ -0,0 +1,234 @@
+          </tr>
+        </tbody>
+      </table>`
+    ),
+  },
+  {
+    label: 'TINY-6664: Assert table width - relative width entire table',
+    expectedWidth: '63.4%',
+    table: (
+      `<table style="border-collapse: collapse; width: 63.352%; height: 258px;" border="1">
+        <tbody>
+          <tr>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+          </tr>
+          <tr>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+          </tr>
+          <tr>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+            <td class="${SEL_CLASS}" style="width: 32.7674%;">&nbsp;</td>
+          </tr>
+        </tbody>
+      </table>`
+    ),
+  },
+  {
+    label: 'TINY-6664: Assert table width - responsive single column',
+    expectedWidth: isIE ? 'auto' : '',
+    table: (
+      `<table style="border-collapse: collapse;" border="1">
+        <tbody>
+          <tr>
+            <td>&nbsp;</td>
+            <td>&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+          <tr>
+            <td>&nbsp;</td>
+            <td>&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+        </tbody>
+      </table>`
+    ),
+  },
+  {
+    label: 'TINY-6664: Assert table width - responsive multiple columns',
+    expectedWidth: isIE ? 'auto' : '',
+    table: (
+      `<table style="border-collapse: collapse;" border="1">
+        <tbody>
+          <tr>
+            <td>&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+          <tr>
+            <td>&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+        </tbody>
+      </table>`
+    ),
+  },
+  {
+    label: 'TINY-6664: Assert table width - responsive entire table',
+    expectedWidth: isIE ? 'auto' : '',
+    table: (
+      `<table style="border-collapse: collapse;" border="1">
+        <tbody>
+          <tr>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+          <tr>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+            <td class="${SEL_CLASS}">&nbsp;</td>
+          </tr>
+        </tbody>
+      </table>`
+    ),
+  },
+];
+
+Arr.each(testCases, (testCase) => {
+  UnitTest.test(testCase.label, assertWidth(testCase));
```

Comment:
Hmm... this will be a little weird, as test names don't normally have JIRA numbers. I guess this is fine for now and hopefully we can come up with something better once we switch to BDD style.